src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -1122,7 +1122,7 @@ static S1 MayWrap(ref int arg)
     }
 ";
             CreateCompilationWithMscorlibAndSpan(text).VerifyDiagnostics(
-                // no diagnostics expected
+            // no diagnostics expected
```

Comment:
this change is minor, but seems also undesirable.

Reply:
probably not a big deal though.

Reply:
I agree this appears to be a bug.